internal/operator-controller/applier/boxcutter.go

```diff
@@ -135,6 +140,54 @@ func (r *SimpleRevisionGenerator) GenerateRevision(
 	return r.buildClusterExtensionRevision(objs, ext, revisionAnnotations), nil
 }
 
+var restrictedMetadata = []string{
```

Comment:
There was a suggestion by Predrag on the ticket to maybe do it as an allow list instead, i.e. just allow:
.spec, .metadata.name, .metadata.namespace, .metadata.annotations, and .metadata.labels. Wdyt?

Reply:
Thanks Todd - I avoid them myself but only mentioned it here because the original suggestion used that terminology.